docs/user/exams/live_statistics.rst

```diff
@@ -0,0 +1,129 @@
+
+
+4.2 Activity log
+^^^^^^^^^^^^^^^^
+
+The header of the box contains the title of the activity log section. In addition, there is a short description of the activities that instructors can see. Similar to the previous section, instructors can click on the arrow to get more details and navigate to the activity log section, which provides detailed statistics about the performed actions and the current progress of each student. The box also contains multiple charts:
+
+.. figure:: instructor/exam_live_statistics_section_overview_activity_annotated.png
+   :alt: The activity log section
+   :align: center
+
+   The activity log section is the second component of the overview section, which provides additional details about the performed actions.
+
+At the top left **(A)**, the chart shows the current number of actions performed by all students during the exam. It provides a overview of how many interactions are performed during the exam. For example, whether there are more interactions at the beginning or more at the end. Additionally, on the top right **(B)**, the bar chart shows the current average number of performed actions. Since students have different habits, and some switch or hand in assignments more often than others, it is interesting to see how they compare to the average.
+
+Last, at the bottom **(C)**, the diagram shows the number of initial actions performed per category, which allows instructors to track precisely when and how many actions students performed. Thus, we provide the possibility to observe how many students start the exam on time or delayed, or whether students often visit the hand-in early page during the exam but then decide to do not so.
+
+
+5. Section: Exercises
+=====================
+
+This section is the second section of a total of three sections. Similar to the overview page, the exercise page contains the four previously mentioned diagrams. In addition to the charts, this page also contains a table that provides details for each exercise.
+
+The table shows the exercise id, the corresponding group id, and the title of the exercise per column. In addition, it highlights the respective type with an icon. In order to provide important information in the most space-saving way, each column is clickable. Additionally, instructors can unfold the column and receive detailed information on the current progress per exercise via three charts.
+
+Like the general exercise overview, an instructor can see the respective initial navigations and first submission per student. In order to see the current activity progress per exercise, instructors can see the current students at this exercise. In addition, we use doughnut charts as opposed to bar charts to provide more variety.
```

Comment:
```suggestion
Like the general exercise overview, an instructor can see the respective initial navigations and first submission per student. In order to see the current activity progress per exercise, instructors can see the current students at this exercise.
```
That last sentence doesn’t add any information to the user, except telling them that we think we were especially innovative and creative when designing the user interface.